packages/realm-server/tests/index.ts

```diff
@@ -4,3 +4,4 @@ import './loader-test';
 import './indexing-test';
 import './module-syntax-test';
 import './permissions/permission-checker-test';
+import './auth-client-test';
```

Comment:
The auth client test wasn't even included in the test suite 😱

Reply:
wow, great catch, I think we should add linting to prevent this happening 😳